elf_version.h

```diff
@@ -0,0 +1,132 @@
+ * number may also be updated if there are other changes contained within a
+ * release that make it desirable for applications to distinguish one release
+ * from another.
+ *
+ * The Major, Minor, and Revision numbers are provided in this header file as
+ * part of the API definition; this macro must expand to a simple integer value,
+ * so that it can be used in simple if directives by the macro preprocessor.
+ *
+ * The Mission Version number shall be set to zero in all officially released
+ * packages, and is entirely reserved for the use of the mission. The Mission
+ * Version is provided as a simple macro defined in the cfe_platform_cfg.h
+ * header file.
+ *
+ * <H2> Version Number Flexibility </H2>
+ *
+ * The major number may increment when there is no breaking change to the API,
+ * if the changes are significant enough to warrant the same level of attention
+ * as a breaking API change.
+ *
+ * The minor number may increment when there have been no augmentations to the
+ * API, if changes are as significant as additions to the public API.
+ *
+ * The revision numbers may increment in implementations where no actual
+ * implementation-specific code has changed, if there are other changes within
+ * the release with similar significance.
+ *
+ * <H2> How and Where Defined </H2>
+ *
+ * The Major, Minor, and Revision components of the version are provided as
+ * simple macros defined in the cfe_version.h header file as part of the API
+ * definition; these macros must expand to simple integer values, so that they
+ * can be used in simple if directives by the macro preprocessor.
+ *
+ * The Mission Version is provided as a simple macro defined in the
+ * cfe_platform_cfg.h header file. As delivered in official releases, these
+ * macros must expand to simple integer values, so that they can be used in
+ * simple macro preprocessor conditions, but delivered code should not prevent a
+ * mission from, for example, deciding that the Mission Version is actually a
+ * text string.
+ *
+ * <H2> Identification of development builds </H2>
+ *
+ * In order to distinguish between development versions, we provide a BUILD_NAME
+ * and a BUILD_NUMBER.
+ *
+ * The BUILDNAME is a unifying monicker that indicates compatibility across the
+ * core Flight System's propduct suite. The build name is a star or
+ * constellation name and changes once per official release. Subsequent build
+ * names starts with the next letter in the 24-character alphabet. In between
+ * official releases, the BUILDNAME will be held constant.
+ *
+ * The BUILDNUMBER reflects the number of commits since the last official
+ * release for each particular component. This integer monotonically increases
+ * for a given BUILDNAME. When a new BUILDNAME is applied, the BUILDNUMBER
+ * resets to zero.
+ */
+
+
+/*
+**  File: elf_version.h
+**
+**  Purpose:
+**  Provide version identifiers for the ELF to cFE Table Converter.
+**
+*/
```

Comment:
I think the single star is preferred format for new code.  Applies to the other comments also (except license header... leave that as a copy/paste for easier substitution in the future).

Reply:
I was about to ask about that.